src/commonMain/kotlin/uuid.kt

```diff
@@ -76,11 +80,11 @@ public class Uuid(val uuid: ByteArray = genUuid()) {
      * - **`5`** – name-based using SHA-1 hashing
      * - **`6`–`15`** – reserved for future extension
      *
-     * Note that the version returned by this function is only meaningful if the [UUID.variant] is
+     * Note that the version returned by this function is only meaningful if the [Uuid.variant] is
      * [RFC 4122](https://tools.ietf.org/html/rfc4122).
      *
      * @return The version number of this [Uuid].
-     * @sample UUIDTest.versions
+     * @sample com.benasher44.uuid.UuidTest.versions
```

Comment:
This doc doesn't seem right (same with variants)

Reply:
IntelliJ (although EAP and probably not a good benchmark right now because latest EAP has some issues) cannot resolve it without for no good reason. 😋